README.md

```diff
@@ -13,32 +13,40 @@ Domains exist in directories starting at [ecommerce](/ecommerce).
 
 ```
 ecommerce/
+├── coupons
```

Comment:
the overall shape of this PR is good to me - it follows the CQRS split, it follows our architecture choices. 

I'd suggest to put it as part of Pricing, though. It feels like it's part of the same vocabulary - the same domain experts would create the coupon strategy as would choose the right pricing.

Also, let's not use code comments (maybe it should be more explicit in the README).

Reply:
:ok_hand: I'll do that